ccip-sdk/src/chain.ts

```diff
@@ -151,6 +151,38 @@ export type UnsignedTx = {
   [ChainFamily.Sui]: never // TODO
 }
 
+/**
+ * Common options for [[generateUnsignedSendMessage]] and [[sendMessage]] Chain methods
+ */
+export type SendMessageOpts = {
+  // Router address on this chain
```

Comment:
```suggestion
  /** Router address on this chain */
```

Use proper JSDoc in comments please